Dockerfile

```diff
@@ -72,8 +74,9 @@ RUN mkdir -p logs/selenium
 # Install required npm dependencies
 
 COPY package.json .
-COPY npm-shrinkwrap.json .
-RUN npm install
+COPY yarn.lock .
+# skips all dev dependencies if NODE_ENV=production.. so..
+RUN yarn install --production=false
```

Comment:
As a stopgap, we need this in order for yarn to install all the `devDependencies`, which is most of them. Ideally we'd sort out what needs to be in `devDependencies` vs `dependencies`.